static/app/views/newsletterConsent.tsx

```diff
@@ -30,9 +30,12 @@ function NewsletterConsent({onSubmitSuccess}: Props) {
         <FieldWrapper stacked={false} hasControlState={false}>
           {t('Pardon the interruption, we just need to get a quick answer from you.')}
         </FieldWrapper>
-        <InputField
+        <RadioField
           name="subscribed"
-          key="subscribed"
+          choices={[
+            ['true', t('Yes, I would like to receive updates via email')],
+            ['false', t("No, I'd prefer not to receive these updates")],
```

Comment:
I validated that using the boolean string values `true` and `false` are correctly parsed by DRFs BooleanField here https://github.com/getsentry/sentry/blob/71e5a8492485d987188018eb020adf13bd3434d6/src/sentry/api/endpoints/user_subscriptions.py#L17